src/templates/dso_api/dynamic_api/api.html

```diff
@@ -502,11 +502,16 @@ <h1>{{ name }}</h1>
               let head = match.substring(0,100).replace(/\n| /g, "");
               let tail = match.substring(match.length-100).replace(/\n| /g, "");
 
-              // Create an element that can be expanded with a click.
-              let result = `<span class="collapsible collapsed" onClick="this.classList.toggle(\'collapsed\')">`;
-              result += `<span class="collapsible_summary">${syntaxHighlight(head)} ... ${syntaxHighlight(tail)}</span>`;
-              result += `<span class="collapsible_content">${syntaxHighlight(match)}</span></span>`;
-              return result;
+              // If coordinates don't describe a point.
+              // Show a summary that can be expanded with a click.
+              if(/^\[\[/.test(head)) {
+                let result = `<span class="collapsible collapsed" onClick="this.classList.toggle(\'collapsed\')">`;
+                result += `<span class="collapsible_summary">${syntaxHighlight(head)} ... ${syntaxHighlight(tail)}</span>`;
+                result += `<span class="collapsible_content">${syntaxHighlight(match)}</span></span>`;
+                return result;
+              } else {
+                match = match.replace(/\n\s*/g, " ");
+              } 
```

Comment:
alternatief: de hele `__collapsed__` er niet op zetten als de coordinate string vrij klein is. Dan hoeft dat hier niet meer speciaal aangepakt te worden.

Reply:
bedoel je `__coordinatestring__`? Kan maar dan blijven de newlines staan in een point. Dus dan wordt een point als nog 4 regels in plaats van 1.

Reply:
goed punt! mooi zo